gems/pending/MiqVm/miq_scvmm_vm.rb

```diff
@@ -0,0 +1,85 @@
+require 'MiqVm/MiqVm'
+require 'Scvmm/miq_scvmm_vm_ssa_info'
+
+class MiqScvmmVm < MiqVm
+  def openDisks(disk_files)
+    part_volumes = []
+
+    $log.debug "MiqScvmmVm::openDisks: no disk files supplied." unless disk_files
+    raise "MiqScvmmVm::openDisks: Uninitialized miq_scvmm" if @ost.miq_hyperv.nil?
+    #
+    # Build a list of the VM's physical volumes.
+    #
+    disk_files.each do |disk_tag, disk_file|
+      disk_info = init_disk_info(disk_file, disk_tag)
+
+      begin
+        disk = MiqDisk.getDisk(disk_info)
+        next if disk.nil?
+      rescue => err
+        $log.error "Couldn't open disk file: #{disk_file} #{err}"
+        $log.debug err.backtrace.join("\n")
+        @diskInitErrors[disk_file] = err.to_s
+        next
+      end
+
```

Comment:
What happened to the setting of `@wholeDisks`? If that's not set, the `umount` may not work properly.
This is another reason I'd prefer this aspect of the code remain common.